tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -6,7 +6,9 @@
 
 from common.utils.tflite_utils import get_tflite_results, save_pb_to_tflite
 from common.utils.tf_utils import save_to_pb
+from common.layer_utils import import_openvino_tokenizers
 
+import_openvino_tokenizers()
```

Comment:
the problem with it is that we will import tokenizers for tests for legacy FE. Can we avoid these changes with import?

Reply:
yes, but current implementation imports openvino_tokenizers in all layers test (onnx, pytorch, etc)